ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -1239,27 +1239,15 @@ periodics:
           - as: e2e
             steps:
               cluster_profile: packet
-              env:
-                DEVSCRIPTS_CONFIG: |
-                  IP_STACK=v4
-                  NETWORK_TYPE=OpenShiftSDN
-              workflow: baremetalds-e2e
+              workflow: baremetalds-e2e-ipv4
           - as: e2e-dualstack
             steps:
               cluster_profile: packet
-              env:
-                DEVSCRIPTS_CONFIG: |
-                  IP_STACK=v4v6
-                  NETWORK_TYPE=OVNKubernetes
-              workflow: baremetalds-e2e
+              workflow: baremetalds-e2e-ovn-dualstack
           - as: e2e-ipv6
```

Comment:
This change needs to be moved to a different PR

Reply:
They are exactly same workflows, latter one is newest, I can open a different PR.